### PR TITLE
Staking era fix

### PR DESCRIPTION
**Pull request by Frenkiee**
Crawler had a bug where staking rewards wouldn't point to their reward destination. Now we are extracting the payee accounts for every reward based on the crawling block. 
To fix the old data there is a populate staking script which can be run through `make net=testnet env=prod script=staking populate`. When running populate script its highly recommended to stop crawling process for the time being!

### CODE REVIEW

File: resources/docker-compose-populate.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - POSTGRES_DATABASE=${DB_NAME} #reefexplorer
+      - POSTGRES_PASSWORD=${DB_PW} #reefexplorer
+      - ENVIRONMENT=${ENVIRONMENT}
+      - NETWORK=${NETWORK}
+
+volumes:
+  populate-modules: {} 
+
+networks:
+  reef-explorer-testnet-prod_reefscan:
```
Comment (frisitano) [inline resources/docker-compose-populate.yml]: This network name is hard coded which is sub-optimal.  You should try to paramaterise this such that environment is passed via command line and appended to network name.

File: docker-compose.yml [hunk 2]
```diff
@@ -68,7 +68,7 @@ services:
       - START_BLOCK_STEP=32
       # with 8 nodes you can process up to 4096 blocks at once but rather keep it at 1024 if server doesn't have a lot of RAM.
       # Currently we support node with 16Gb of RAM. To change the capacity go in `./crawler/package.json` and change --max-old-space-size=16384 to desired size
-      - MAX_BLOCKS_PER_STEP=32
```
Comment (frisitano) [inline docker-compose.yml]: why has this been increased?

Comment (Frenkiee) [reply docker-compose.yml]: when crawling data on localhost I prefer using a bigger step for less frontend lag

File: resources/docker-compose-populate.yml (hunk 1 above)
Comment (Frenkiee) [reply resources/docker-compose-populate.yml]: done

**Comment by frisitano** [comment]
can you test this please?  Deploy new crawler image to dev cluster and then backfill?

**Pull request merged**